R/expect_s4_class_linter.R

```diff
@@ -26,19 +26,18 @@ expect_s4_class_linter <- function() {
   # require 2 expressions because methods::is(x) alone is a valid call, even
   #   though the character output wouldn't make any sense for expect_true().
   xpath <- "
-  //SYMBOL_FUNCTION_CALL[text() = 'expect_true']
-    /parent::expr
+  parent::expr
     /following-sibling::expr[1][count(expr) = 3 and expr[1][SYMBOL_FUNCTION_CALL[text() = 'is']]]
     /parent::expr[not(SYMBOL_SUB[text() = 'info' or text() = 'label'])]
   "
 
   Linter(linter_level = "expression", function(source_expression) {
-    xml <- source_expression$xml_parsed_content
-
     # TODO(michaelchirico): also catch expect_{equal,identical}(methods::is(x), k).
```

Comment:
Just noticed this TODO again, checked on GitHub... there are plenty of hits but it's not epidemic:

https://github.com/search?q=lang%3AR%20%2Fexpect_(equal%7Cidentical)%5B(%5D%5Cs*(methods%3A%3A)%3Fis%5B(%5D%2F&type=code

Reply:
#2423

Reply:
```suggestion
    # TODO(#2423): also catch expect_{equal,identical}(methods::is(x), k).
```